python/ray/air/callbacks/wandb.py

```diff
@@ -87,15 +88,16 @@ def _clean_log(obj: Any):
         return obj
 
     # Else
+    from wandb.util import json_dumps_safer
+
     try:
+        # This is what wandb uses internally. If we cannot dump
+        # an object using this method, wandb will raise an exception.
+        json_dumps_safer(obj)
```

Comment:
Can we use `wandb.util.json_dumps_safer` or does this not work without the import?
Would be great to avoid local imports